server/sonar-server/src/main/java/org/sonar/server/ce/ws/CancelAction.java

```diff
@@ -42,7 +41,7 @@ public CancelAction(UserSession userSession, CeQueue queue) {
   @Override
   public void define(WebService.NewController controller) {
     WebService.NewAction action = controller.createAction("cancel")
-      .setDescription("Cancels a pending task. Requires system administration permission. In-progress tasks can not be canceled.")
+      .setDescription("Cancels a pending task. Requires system administration permission. In-progress tasks cannot be canceled.")
```

Comment:
description should be fixed, root is now required, not system administration... or is that the name we give to root?